examples/tutorials/mvdr_tutorial.py

```diff
@@ -1,32 +1,46 @@
 """
-MVDR with torchaudio
-====================
+MVDR Beamforming with torchaudio
+================================
 
 **Author** `Zhaoheng Ni <zni@fb.com>`__
 
 """
 
+
 ######################################################################
-# Overview
-# --------
+# 1. Overview
```

Comment:
is there a way to generate section numbering automatically? would be nice to not have to worry about it and to be able to easily apply it to all tutorials

Reply:
I think so. In our contributing.md file it suggests using https://gist.github.com/chsasank/7218ca16f8d022e02a9c0deb94a310fe to convert the ipynb notebook to sphinx format. We can extend it to support auto-indexing.

Reply:
Tutorials are not necessarily authored with IPython Notebook. (I directly write these from editor) so that's not a viable option. Would be nice to have auto numbering but sphinx-gallery only does the rendering of source, so I do not think there is a good way to automate it.

Reply:
I started numbering to make sections and subsections flat because subsection was hidden by default. But now that is resolved by #2397, I am okay with removing numbering entirely or at least from subsections.